mozci/push.py

```diff
@@ -1190,34 +1190,28 @@ def classify_regressions(
         else:
             all_groups = push_groups
 
-        groups_relevant_failure = {
+        groups_consistent_failure = {
             g.name
             for g in all_groups
-            if (
-                cross_config_counts is not None
-                and g.is_cross_config_failure(cross_config_counts[1])
-            )
-            or (
-                consistent_failures_counts is not None
-                and g.is_config_consistent_failure(consistent_failures_counts[1])
+            if g.is_consistent_failure(
+                cross_config_counts[1] if cross_config_counts is not None else None,
+                consistent_failures_counts[1]
+                if consistent_failures_counts is not None
+                else None,
             )
         }
 
-        groups_non_relevant_failure = {
+        groups_non_consistent_failure = {
             g.name
             for g in all_groups
             if g.status != Status.PASS
```

Comment:
Yeah makes sense, I think it was harder earlier to see this because of the conditions being in the inline if. Now with the refactoring it is clear!

Reply:
Oh actually no, we can't because the parameters are slightly different.
For marking as "real regression", to be more conservative, we consider a failure to be failing consistently if it failed at least three times (so a single failing retrigger is not enough).
For marking as "intermittent", we consider a failure to be non cross-config if there at least a retrigger in which it didn't fail.